src/ui/routes/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+	import { clusterStore } from '$lib/store';
+
+	import { goto } from '$app/navigation';
+	import Spinner from '$lib/components/spinner.svelte';
+	import bigZarf from '@images/zarf-bubbles-right.png';
+	import { Button } from '@ui';
+	import Hero from '$lib/components/hero.svelte';
+</script>
+
+{#if $clusterStore}
+	{#if $clusterStore.hasZarf}
+		{goto(`/packages`, { replaceState: true })}
+	{:else}
+		<Hero>
+			<img src={bigZarf} alt="Zarf logo" id="zarf-logo" width="40%" />
+
+			<div class="hero-text">
+				<h1 class="hero-title">No Active Zarf Clusters</h1>
+
+				{#if $clusterStore.reachable && $clusterStore.distro !== 'unknown'}
+					<h2 class="hero-subtitle">
+						A {$clusterStore.distro} cluster was found, click initialize cluster to initialize it now
+						with Zarf.
+					</h2>
+				{:else}
+					<h2 class="hero-subtitle">
+						Click initialize cluster to install the Init Package and deploy a new cluster.
+					</h2>
+				{/if}
+			</div>
+
+			<Button variant="raised" color="primary" href="/initialize/configure" id="init-cluster"
+				>Initialize Cluster</Button
```

Comment:
This is a really weird way to break up the <>

Reply:
I hatez it...it's what prettier is doing (I think) and it's 🤮. Maybe we can tweak prettier defaults?